docs/modules/ROOT/pages/usage.adoc

```diff
@@ -131,4 +134,9 @@ In `spec.druid` you specify the `name` and `namespace` of your Druid cluster.
 
 Once the database is initialized, the connection will be added to the cluster by the operator. You can see it in the user interface under Data > Databases:
 
-image::superset-databases.png[Superset databases showing the connected Druid cluster]
\ No newline at end of file
+image::superset-databases.png[Superset databases showing the connected Druid cluster]
+
+== Monitoring
+
+The managed HBase instances are automatically configured to export Prometheus metrics. See
+xref:home::monitoring.adoc[] for more details.
```

Comment:
This link seems to be broken/incorrect.

Reply:
This link points to https://docs.stackable.tech/home/monitoring.html and works only together with the base documentation.